### PR TITLE
STM32F0 and nRF52832 support

### PR DESCRIPTION
**Pull request by jklmnn**
- Support for nRF52832 (#61)
 - Support for STM32F051 (#64)
 - Migration from Travis to GitHub Actions
   To save time the Genode Docker container no longer comes with prepared ports. However on Travis some ports fail to prepare (cf. https://github.com/travis-ci/travis-ci/issues/3692). This is no problem on GitHub Actions.

### CODE REVIEW

File: platform/nrf52/bluefruit_feather/componolit-runtime-board.adb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+   procedure Log (S : String) is null;
+
+   procedure Halt_On_Error
+   is
+      procedure Wait;
+      procedure Wait
+      is
+      begin
+         for I in Integer range 0 .. 1000000 loop
+            null;
```
Comment (senier) [inline platform/nrf52/bluefruit_feather/componolit-runtime-board.adb]: I wonder if (or why not) this is optimized out when compiling with high optimization? Maybe we need something like [`pragma Inspection_Point (I)`](https://www.adaic.org/resources/add_content/standards/12rm/html/RM-H-3-2.html) here?

**Review by senier — CHANGES_REQUESTED** [review]

File: platform/nrf52/bluefruit_feather/componolit-runtime-board.adb (hunk 1 above)
Comment (jklmnn) [reply platform/nrf52/bluefruit_feather/componolit-runtime-board.adb]: Currently there are no optimizations enabled, so it is not removed. I still added the inspection point in https://github.com/Componolit/ada-runtime/pull/65/commits/f0c64e5aa731a21f757572f140a34c0869bd38a9.

**Comment by jklmnn** [comment]
As a general notice, the ARM support for both chips is only scarcely tested. So we should treat (and name) it as such until we have tested and used it more thoroughly.

**Comment by RREE** [comment]
Deducing from the AVR compiler I can confirm that a loop with no side effects is completely removed. An inspection point or reading from a volatile variable inhibits the removal.Regards    Rolf -- Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Am 02.04.20, 17:41 schrieb JK <notifications@github.com>:

   
  @jklmnn commented on this pull request. 
   
  In platform/nrf52/bluefruit_feather/componolit-runtime-board.adb: 
  > +   is
+   begin
+      Drivers.GPIO.Configure (Blue, Drivers.GPIO.Port_Out);
+      Drivers.GPIO.Write (Blue, Drivers.GPIO.High);
+   end Initialize;
+
+   procedure Log (S : String) is null;
+
+   procedure Halt_On_Error
+   is
+      procedure Wait;
+      procedure Wait
+      is
+      begin
+         for I in Integer range 0 .. 1000000 loop
+            null;
 
  Currently there are no optimizations enabled, so it is not removed. I still added the inspection point in f0c64e5.

**Comment by senier** [comment]
> Deducing from the AVR compiler I can confirm that a loop with no side effects is completely removed. An inspection point or reading from a volatile variable inhibits the removal.Regards Rolf

@RREE Thanks for confirming!

**Review by senier — APPROVED** [review]

**Pull request merged**